app/views/wheres/index.html.erb

```diff
@@ -15,6 +15,10 @@
           <td><%= link_to user.name, user_wheres_path(user) %></td>
           <td><%= user.team_name %></td>
           <td data-timestamp='<%= where.sent_at.try(:to_i) %>'>
+            <%= where.subject %>
+            <% if where.subject.present? && where.body.present? %>
+              <hr />
```

Comment:
Can we distinguish the subject by its font style instead (e.g. bold)? The rows of this table are striped as well, and I think having `<hr>`s inside the rows intermittently might not make it easy to separate the components at a glance.

Reply:
done!